### PR TITLE
Adding month calendar unit tests

### PR DESCRIPTION
**Pull request by lisoenot**
Fixes #

## Proposed changes

- Add unit tests for MonthCalendarAccessibleObject class and its children

## Customer Impact

- No

## Regression? 

- No

## Risk

- No

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/5147)

### CODE REVIEW

**Comment by RussKie** [comment]
Please do a rebase on top of the latest main

**Review by vladimir-krestov — APPROVED** [review]

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs [hunk 1]
```diff
@@ -32,5 +32,50 @@ public void CalendarTodayLinkAccessibleObject_Description_ReturnsExpected()
...
         }
+
+        [WinFormsFact]
+        public void CalendarPreviousButtonAccessibleObject_GetChildId_ReturnsExpected()
+        {
+            using MonthCalendar control = new();
+
+            control.CreateControl();
+
+            MonthCalendarAccessibleObject controlAccessibleObject = (MonthCalendarAccessibleObject)control.AccessibilityObject;
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs:43]: lets keep it consistent with 'var' usage.  If type is evident from reading code ( in this case, type cast is helping  that way) we go with 'var'.

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs [hunk 2]
```diff
@@ -32,5 +32,50 @@ public void CalendarTodayLinkAccessibleObject_Description_ReturnsExpected()
...
+
+            Assert.Equal(4, actual);
+            Assert.True(control.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CalendarPreviousButtonAccessibleObject_GetChildId_ReturnsExpected_IfCalendarsAccessibleObjectsIsNull()
+        {
+            using MonthCalendar control = new();
+            MonthCalendarAccessibleObject controlAccessibleObject = (MonthCalendarAccessibleObject)control.AccessibilityObject;
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs:56]: same here and other places.

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs [hunk 3]
```diff
@@ -32,5 +32,50 @@ public void CalendarTodayLinkAccessibleObject_Description_ReturnsExpected()
...
+            using MonthCalendar control = new();
+
+            control.CreateControl();
+
+            MonthCalendarAccessibleObject controlAccessibleObject = (MonthCalendarAccessibleObject)control.AccessibilityObject;
+            CalendarTodayLinkAccessibleObject todayLinkAccessibleObject = new(controlAccessibleObject);
+
+            int actual = todayLinkAccessibleObject.GetChildId();
+
+            Assert.Equal(4, actual);
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendar.CalendarTodayLinkAccessibleObjectTests.cs:48]: Can you add comment on constants used here and below on why and how that number is expected.

**Review by RussKie — COMMENTED** [review]
👍

**Pull request merged**